src/applications/simple-forms/21-10210/pages/claimantType.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { CLAIMANT_TYPES, CLAIM_OWNERSHIPS } from '../definitions/constants';
+
+export default {
+  uiSchema: {
+    claimantType: {
+      'ui:widget': 'radio',
+      'ui:options': {
+        hideIf: formData => formData.claimOwnership === undefined,
+        updateSchema: (formData, schema, uiSchema) => {
+          const { claimOwnership } = formData;
+          const uiSchemaCopy = { ...uiSchema };
+          let title;
+          switch (claimOwnership) {
+            case CLAIM_OWNERSHIPS.SELF:
+              title = (
+                <span className="vads-u-font-family--serif vads-u-font-size--h3 vads-u-font-weight--bold">
+                  Which of these descriptions best describes you?
+                </span>
+              );
+              uiSchemaCopy['ui:options'].labels = {
```

Comment:
something to look out with this pattern is that we're also modifying the original copy here (since x = { ...y } is a shallow copy). 
but in this case it seems it doesn't matter and is all good 👍

Reply:
Fixed.  Using lodash cloanDeep now instead.